modules/seqexec/server/src/main/scala/seqexec/server/InstrumentControllerSim.scala

```diff
@@ -45,7 +45,7 @@ sealed trait InstrumentControllerSim[F[_]] {
 }
 
 object InstrumentControllerSim {
-  private final class InstrumentControllerSimImpl[F[_]: Sync: Timer](name: String, useTimeout: Boolean) extends InstrumentControllerSim[F] {
+  private final class InstrumentControllerSimImpl[F[_]](name: String, useTimeout: Boolean)(implicit val F: Sync[F], T: Timer[F]) extends InstrumentControllerSim[F] {
```

Comment:
What is the advantage of this style over the previous version ?

Reply:
It is a small thing and I'm not consistent but rather than doing `Sync[F].delay` I can just do `F.delay`